static/app/components/group/sentryAppExternalIssueForm.tsx

```diff
@@ -27,6 +27,7 @@ function SentryAppExternalIssueForm({
   sentryAppInstallation,
 }: Props) {
   const contentArr = getStacktraceBody(event);
+  const isFeedback = (group.issueCategory as string) === 'feedback';
```

Comment:
Is there an enum for this?

Reply:
no, it's defined as a hardcoded string on the feedback type because making an enum means making a bunch of other configs/big code changes that haven't done yet

Reply:
https://github.com/getsentry/sentry/blob/e15cc72dc2ba86bb57680df027ecd1257bda481f/static/app/utils/feedback/types.tsx#L8